packages/mobile/src/web3/contracts.ts

```diff
@@ -13,6 +14,7 @@ import { Provider } from 'web3/providers'
 const tag = 'web3/contracts'
 
 export const web3: Web3 = getWeb3()
+export const contractKit = newKitFromWeb3(web3)
```

Comment:
Will this be okay when toggling to zero sync?

Reply:
We may need to update the contractkit provider as well, but I can deal with that in the zeroSync PR. Thanks for flagging this

Reply:
Follow up on this, tested after merging in the ZeroSync PR and everything works as expected